pkg/gitreceive/git/sha.go

```diff
@@ -0,0 +1,37 @@
+package git
+
+import (
+	"fmt"
+	"regexp"
+)
+
+const (
+	// this constant represents the length of a shortened git sha - 8 characters long
+	shortShaIdx = 8
+	fullShaLen  = 40
```

Comment:
Fixed in https://github.com/deis/builder/commit/8e8c0a6d7c14997aaf919fe92170dd4f613a718d